sdk/communication/Azure.Communication.MediaComposition/src/autorest.md

```diff
@@ -7,11 +7,19 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 title: Media Composition
-tag: package-2022-07-16-preview1
-generation1-convenience-client: true
+tag: package-2022-07-16-preview
 model-namespace: false
+generation1-convenience-client: true
 payload-flattening-threshold: 10
-require:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e2de3bd42679ab281a61c52b4a24252d5394449a/specification/communication/data-plane/MediaComposition/readme.md
+input-file:
+    - mediaComposition.json
```

Comment:
Instead of having a local copy, can we just push a new commit to https://github.com/Azure/azure-rest-api-specs/tree/feature/communication-media-composition?

Reply:
Will do in another PR!